modules/apps/27-interchain-accounts/controller/keeper/handshake.go

```diff
@@ -79,6 +79,10 @@ func (k Keeper) OnChanOpenAck(
 		return sdkerrors.Wrapf(icatypes.ErrUnknownDataType, "cannot unmarshal ICS-27 interchain accounts metadata")
 	}
 
+	if activeChannelID, found := k.GetOpenActiveChannel(ctx, portID); found {
+		return sdkerrors.Wrapf(porttypes.ErrInvalidPort, "existing active channel %s for portID %s", activeChannelID, portID)
```

Comment:
I think the base error could be something else? Not sure the port is technically invalid